### ISSUE TITLE
testCreateUploadedFileWithString deprecated

### ISSUE THREAD

**Issue by MarcHagen** [http-interop/http-factory-tests#30]
Has of the latest version of PSR-17 UploadedFileFactoryInterface no longer accepts string but only SteamInterface. Therefore this test should be removed.

PSR-17 snipped:
```php
    public function createUploadedFile(
        StreamInterface $stream,
        int $size = null,
        int $error = \UPLOAD_ERR_OK,
        string $clientFilename = null,
        string $clientMediaType = null
    ): UploadedFileInterface;
```

**Comment by Nyholm** [comment]
Thank you. This was fixed by #32

**Comment by MarcHagen** [comment]
Thx!
